WalletWasabi/Hwi/Models/HwiEnumerateEntry.cs

```diff
@@ -62,10 +62,12 @@ public bool IsOfflinePsbtWorkflowCompatible()
 			HardwareWalletModels.Coldcard => true,
 			HardwareWalletModels.Ledger_Nano_S or HardwareWalletModels.Ledger_Nano_X or HardwareWalletModels.Ledger_Nano_S_Plus => false,
 			HardwareWalletModels.Trezor_1 => false,
-			HardwareWalletModels.Trezor_T => true,
+			HardwareWalletModels.Trezor_T => false,
```

Comment:
Was this value wrong and it got corrected?

Do we still need `IsOfflinePsbtWorkflowCompatible`?

Reply:
Yes, I need it in upcoming features